File: katt/iterated/submissions/accepted/iterated.py
#!/usr/bin/env python2
import stub

def init(n):
    g = [False] * n
    cor = stub.guess(g)
    for i in range(n):
        g[i] = 1
        cor2 = stub.guess(g)
        if cor2 < cor: g[i] = 0
        else: cor = cor2

File: katt/Contest3/iterated/submissions/accepted/iterated.java
class iterated {
	public static void init(int N) {
		boolean[] g = new boolean[N];
		int cor = stub.lib.guess(g);
		for (int i = 0; i < N; i++) {
			g[i] = true;
			int cor2 = stub.lib.guess(g);
			if (cor2 < cor) g[i] = false;
			else cor = cor2;
		}
	}
}
